13-microbiome.Rmd

```diff
@@ -0,0 +1,63 @@
++ functional activity (such as metatranscriptome sequencing)
+
+Other “omics” technologies are now playing an increasing role in microbiome research, such as:
++ mass-spectrometry-based metabolomics, which provides profiles of small molecule metabolites in an environment.
++ metaproteomics which provides more detailed descriptions of functional activities of microbes (and their hosts, if applicable).
+
+ As a result, bioinformatics software tools are essential to microbiome research. For many microbiome researchers, bioinformatics is an intimidating and challenging aspect of their projects.
+
+
+## Microbiome Analysis with QIIME 2
+QIIME 2 is an all in one bioinformatics microbiome analysis platform. This platform allows for users to go from sequenced microbiome data to publication ready visualizations. The original QIIME, now referred to as QIIME 1, was published in 2010 [@Caporaso2010] and has been cited tens of thousands of times in the primary literature. QIIME 2, which was published in July of 2019 [@Bolyen2019], succeeded QIIME 1 on 1 January 2018. QIIME 2 is better than QIIME 1 in all ways, and QIIME 1 is no longer actively supported. If you have previously used QIIME 1, you should invest time in learning and switching to QIIME 2. If you’re new to QIIME, start with QIIME 2. (When I refer to QIIME in this book, without specifying whether I’m referring to QIIME 1 or QIIME 2, I’m referring to the platform generally.)
+
+QIIME 2 has large and growing user and developer communities, and these communities make QIIME 2 possible. The epicenter of the community is the QIIME 2 Forum. The forum is primarily known as a place where users can get technical support with QIIME 2 for no charge. Developers of QIIME 2 moderate the forum, and typically respond to technical support questions within a couple of business days. The forum is also a great place to discuss general topics in microbiome bioinformatics, or microbiome research methods generally. There are many active discussions on these topics on the forum. Keeping up with the discussions on the forum is a great way to learn about current topics in microbiome research methods. There’s also a free job board on the forum - you can use the forum to find jobs, or post your own job ads there to find employees who are well-versed in QIIME 2 and other bioinformatics tools. If you’re not already a member of the QIIME 2 Forum, you should consider joining. It’s a great way for you to get help, and as you develop your QIIME 2 skills helping others on the forum is a great way to reenforce your learning and to get involved in the community.
+
+[Here](https://gregcaporaso.github.io/q2book/front-matter/preface.html) is a high-level introduction to microbiome analysis using QIIME 2. This introduction will go over common methods, metrics and approaches used for microbiome science.
+So grab a cup of your favorite hot beverage and let’s get started! ☕
+
+## More resources
```

Comment:
```suggestion
## More resources
```

Any additional reading you'd recommend in this section? Otherwise we can just delete this header.